helm/aws-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,39 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aws-operator.name" -}}
+{{- default .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "aws-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "aws-operator.labels" -}}
+helm.sh/chart: {{ include "aws-operator.chart" . }}
+{{ include "aws-operator.selectorLabels" . }}
+app.giantswarm.io/branch: {{ .Values.project.branch }}
+app.giantswarm.io/commit: {{ .Values.project.commit }}
+app.kubernetes.io/name: {{ include "aws-operator.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- if .Chart.AppVersion }}
+app.giantswarm.io/version: {{ .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
```

Comment:
I guess one of them is enough :D Also this is going to be set by architect and required by chart linting introduced in ~8.0.2 architect orb.

```suggestion
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Reply:
Done.